src/ensembl/production/metadata/updater/base.py

```diff
@@ -54,3 +57,24 @@ def get_or_new_source(self, meta_session, db_type):
             )
             meta_session.add(dataset_source)  # Only add a new DatasetSource to the session if it doesn't exist
         return dataset_source
+
+    def update_attributes(self, dataset, attributes, session):
+        genebuild_dataset_attributes = []
```

Comment:
Probably should change the name from "genebuild_dataset_attributes" with it out here and able to be used by anything, even non-core databases.